osf/external/chronos.py

```diff
@@ -174,6 +174,18 @@ def submit_manuscript(self, journal, preprint, submitter):
         if ChronosSubmission.objects.filter(journal=journal, preprint=preprint).exists():
             raise ValueError('{!r} already has an existing submission to {!r}.'.format(preprint, journal))
 
+        # 1 = draft, 2 = submitted, 3 = accepted, 4 = published
```

Comment:
minor rewrite
```python
def submit_manuscript(self, journal, preprint, submitter):
    submission_qs = ChronosSubmission.objects.filter(preprint=preprint) 
    if submission_qs.filter(journal=journal).exists():
        raise ValueError('{!r} already has an existing submission to {!r}.'.format(preprint, journal))

    # 1 = draft, 2 = submitted, 3 = accepted, 4 = published
    # Disallow submission if the current preprint has submissions that are submitted, accepted or publishes
    # regardless of journals
    if submission_qs.filter(status=1).exists():
        raise ValueError('Cannot submit because a drafted submission exists')
    if submission_qs.filter(status=2).exists():
        raise ValueError('Cannot submit because a pending submission exists')
    if submission_qs.filter(status__in=[3, 4]).exists():
        raise ValueError('Cannot submit because your submission was accepted or published')

    body = ChronosSerializer.serialize_manuscript(journal.journal_id, preprint)
    body['USER'] = ChronosSerializer.serialize_user(submitter)
```